janusgraph-dist/README.md

```diff
@@ -15,7 +15,7 @@ SNAPSHOT or a release tag.
 
 This command writes one archive:
 
-* janusgraph-dist/janusgraph-dist-hadoop-2/target/janusgraph-$VERSION-hadoop2.zip
+* janusgraph-dist/target/janusgraph-$VERSION.zip
```

Comment:
While this is a much better name and path for the distribution, this change will totally throw me off for a good chunk of time.